resources/lang/en/layout.php

```diff
@@ -79,6 +79,7 @@
             '_' => 'help',
             'getFaq' => 'faq',
             'getRules' => 'rules',
+            'getAbuse' => 'report abuse',
```

Comment:
Translation keys are sorted. Also not sure about calling it `getAbuse` :thinking: (and I'm not sure why this entire section is still using `get` prefix either)